README.md

```diff
@@ -1 +1,72 @@
-# Leader-Board
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project : Leaderboard APP
+
+> The leaderboard App is designed to display scores submitted by different players. It also allows submitting a player's score, where the data is preserved thanks to the external Leaderboard API service. ES6 features have been properly implemented in this app. Connecting with APIs successfully by proper utilization of async and await..
+
+
+
+
+
+## Built With
+
+#### Major languages:
+>- [x] HTML5
+>- [x] CSS3
+>- [x] Javascript
+
+#### Technologies used
+>- [x] Linters
+>- [x] ES 6
+>- [x] Webpack
+>- [x] Gitflow
+
+## Live Demo (if available)
+
+[Live Demo Link](https://akos29.github.io/To-do-list/)
```

Comment:
[OPTIONAL] Kindly update the live demo link to show the correct project since the available link points to a different project. :wink: